src/WebJobs.Script/Description/Rpc/WorkerFunctionDescriptorProvider.cs

```diff
@@ -21,13 +21,15 @@ internal class WorkerFunctionDescriptorProvider : FunctionDescriptorProvider
     {
         private readonly ILoggerFactory _loggerFactory;
         private IFunctionDispatcher _dispatcher;
+        private string _currentRuntimelanguage;
 
-        public WorkerFunctionDescriptorProvider(ScriptHost host, ScriptJobHostOptions config, ICollection<IScriptBindingProvider> bindingProviders,
+        public WorkerFunctionDescriptorProvider(ScriptHost host, string currentRuntimelanguage, ScriptJobHostOptions config, ICollection<IScriptBindingProvider> bindingProviders,
             IFunctionDispatcher dispatcher, ILoggerFactory loggerFactory)
             : base(host, config, bindingProviders)
         {
             _dispatcher = dispatcher;
             _loggerFactory = loggerFactory;
+            _currentRuntimelanguage = currentRuntimelanguage;
```

Comment:
Fix camel casing. Also, can we call this just runtimeLanguage rather than "current" here and throughout?

Reply:
Updated all of them to workerRuntime